includes/restapi/class-notification-controller.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * REST API controller to get and set notifications.
+ *
+ * @package wordpress/wp-feature-notifications
+ */
+
+namespace WP\Notifications\REST;
+
+use WP_Error;
+use WP_REST_Controller;
+use WP_REST_Request;
+use WP_REST_Response;
+use WP_REST_Server;
+
+/**
+ * REST API Controller class
+ */
+class Notification_Controller extends WP_REST_Controller {
+
+	/**
+	 * Namespace for the REST endpoint.
+	 *
+	 * @type string
+	 */
+	const NAMESPACE = 'wp/v2';
+
+	/**
+	 * Base for notification REST routes.
+	 *
+	 * @type string
+	 */
+	const NOTIFICATION_BASE = 'notifications';
+
+	public function __construct() {
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register REST routes
+	 *
+	 * @return void
+	 */
+	public function register_routes() : void {
+		register_rest_route(
+			self::NAMESPACE,
+			self::NOTIFICATION_BASE,
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'args'                => array(),
+				'callback'            => array( $this, 'get_notifications' ),
+				'permission_callback' => array( $this, 'authenticate' ),
+			),
+			false,
```

Comment:
```suggestion
			false
```
@Sephsekla if you could remove this comma that causes the failing action, we can merge this commit, so I start the rebase with the js branch with also this commit